src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,83 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  closeTodo: () => void,
+  selectedTodo: Todo | null,
+};
+
+export const TodoModal: React.FC<Props> = ({ selectedTodo, closeTodo }) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  const fetchData = async () => {
+    try {
+      const currentUser = await getUser(selectedTodo?.userId || 0);
+
+      setUser(currentUser);
+    } catch (error) {
+      if (error instanceof Error) {
+        // eslint-disable-next-line no-console
+        console.warn(error.message);
```

Comment:
you may handle the error in a similar way as in App.tsx - create `isError` state and show some error message in case it's `true`